app/Main.hs

```diff
@@ -148,20 +160,36 @@ main :: IO ()
 main = do
   env <- getEnvironment
   opts <- execParser (optionsInfo env)
+  manager <- getManager opts
 
-  eres <- runExceptT $ runReaderT vaultEnv (env, opts)
+  eres <- runExceptT $ runReaderT vaultEnv (ExecCtx env opts manager)
```

Comment:
s/eres/eitherNewEnvironment

Reply:
Since it is just a temporary binding that we immediately throw into a case and isn't needed any further, I've enabled `LambdaCase`, which works around the problem. In this instance, I'd argue the naming is just noise.

The extension doesn't do much, but it alleviates the need for giving a name to the binding. I'd say it should be in our standard set of extensions to allow/enable. More on [this idiom](http://storm-country.com/blog/LambdaCase)